packages/compass-crud/src/components/crud-toolbar.tsx

```diff
@@ -75,6 +75,11 @@ const docsPerPageOptionStyles = css({
   width: spacing[1600] + spacing[300],
 });
 
+const loaderContainerStyles = css({
+  paddingLeft: 8,
```

Comment:
We normally avoid using numbers directly and use `spacing` from compass-components. In this case it would be `spacing[200]`